tests/e2e-tests/test-devicestate.sh

```diff
@@ -0,0 +1,98 @@
+# Insert file header
+
+# Insert function header
+function cleanup() {
+	echo Starting cleanup
+	
+	local armToken=$1
+	local apiToken=$2
+	local device_id=$3
+	local token_id=$4
+
+	# Clean up device if it exists
+	local device_exists=$(curl -X GET -H "Authorization:$apiToken" https://e2etest-iotc-iiot-asset-app.azureiotcentral.com/api/preview/devices/${device_id}  2>/dev/null | jq -r '.id')	
+	if [ "$device_exists" == "$device_id" ];
+	then
+		echo Device ${device_id} exists
+		echo Delete device ${device_id}
+		curl -X DELETE -H "Authorization:Bearer $armToken" https://e2etest-iotc-iiot-asset-app.azureiotcentral.com/api/preview/devices/${device_id} 2>/dev/null
+	else
+		echo Device ${device_id} does not exist
+	fi;
+
+	# Clean up API token if it exists
+	local apiToken_exists=$(curl -X GET -H "Authorization:$apiToken" https://e2etest-iotc-iiot-asset-app.azureiotcentral.com/api/preview/apiTokens/${token_id} 2>/dev/null | jq -r '.id')
+	if [ "$apiToken_exists" == "$token_id" ];
+	then
+		echo API token ${token_id} exists
+		echo Delete API token ${token_id}
+		curl -X DELETE -H "Authorization:Bearer $armToken" https://e2etest-iotc-iiot-asset-app.azureiotcentral.com/api/preview/apiTokens/${token_id} 2>/dev/null
+	else
+		echo API token ${token_id} does not exist
+	fi;
+
+	echo Completed cleanup
+}
+
+# Azure IoT DDE team subscription
+subscription=377c3343-75bb-4244-98a3-0fb84a830c4b
```

Comment:
is this a security concern to check in our subscription id?

Reply:
i don't think so, because folks outside of our team have no access to it.